airflow/api_connexion/openapi/v1.yaml

```diff
@@ -178,30 +183,46 @@ info:
     We follow the error response format proposed in [RFC 7807](https://tools.ietf.org/html/rfc7807)
     also known as Problem Details for HTTP APIs.  As with our normal API responses,
     your client must be prepared to gracefully handle additional members of the response.
+
```

Comment:
Mostly I don't care about whitespace, but this is a markdown document in the YAML doc, so I don't have syntax highlighting, so a large block of text is very bad to read.